tox.ini

```diff
@@ -68,3 +68,31 @@ commands =
         --cache-dir="{toxworkdir}/mypy_cache"  \
         {tty:--pretty:}                        \
         {posargs:pydoctor}
+
+
+[testenv:apidocs]
+description = Build only the API documentation
+
+whitelist_externals =
+    cat
+    test
+commands =
+    pydoctor \
+    --add-package=pydoctor \
+    --project-name=pydoctor \
+    --project-url=https://github.com/twisted/pydoctor/ \
+    --html-viewsource-base=https://github.com/twisted/pydoctor/tree/master \
```

Comment:
Source links that point to a branch, in particular the master branch, will get outdated. We should use a tag or commit ID instead.

Reply:
This was moved to https://github.com/twisted/pydoctor/pull/256

This is an tox env for manual testing...so I went with just master as the default link.

This tox env is not designed to be used for production / public site